### DISCUSSION TITLE
Multiplayer Infinitely Loads

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#14336] [Q&A] [answered]
The title is pretty self-explanatory; nothing loads no matter how long I wait upon entering the multiplayer screen. This problem used to occur on the playlists screen until the newest update (2021.815.0)
While the game doesn't throw any errors, I was able to find some and isolated them in this [network.log](https://github.com/ppy/osu-web/files/6989338/network.log). Not an urgent issue (and tbh I'm probably just missing something obvious); I'm mainly putting this here for anyone else that might have the same problem.
PS This is my first time submitting an issue, so apologies if I did anything wrong.

**Comment by peppy** [comment, marked as answer]
You are getting a 404 error. Please try loading https://spectator.ppy.sh in your browser. It will likely give you a solution.

**Reply by zachmanthethird** [reply]
You're right. spectator.ppy.sh gives me a 404 error. I can see osu.ppy.sh without any issues. Do you think it's my firewall?

**Reply by peppy** [reply]
I'd guess that's the case. It should give you a download instead. We aren't doing anything special so it seems like something at your end (or your ISP?).

**Reply by zachmanthethird** [reply]
Still working on connecting, but it's obviously a problem on my side. Thanks for the help :smiley:

**Reply by PercyDan54** [reply]
I'm also getting a 404 from that website in the browser, but I can play multiplayer normally.

**Reply by zachmanthethird** [reply]
After much debate with my ISP, they finally found the root of the problem. Despite seeing https://osu.ppy.sh in my browser, it was impossible to ping any ppy services via the command prompt. Loading osu!lazer via a cellular connection fixed everything :smiley:

**Reply by peppy** [reply]
Thanks for the update. Did your ISP give you any explanation as to why this happened? Would be good to know in case other similar issues arise.
